runtime/test/planner-tests.js

```diff
@@ -293,7 +293,58 @@ describe('ConvertConstraintsToConnections', async () => {
     let strategizer = {generated: [{result: recipe, score: 1}]};
     let cctc = new ConvertConstraintsToConnections({pec: {}});
     let {results} = await cctc.generate(strategizer);
-    assert(results.length == 0);
+    assert.equal(0, results.length);
+  });
+
+  it('only input handles with mapped handle are resolved', async () => {
```

Comment:
Nit. 'it' should be part of the sentence.

Reply:
updated.
Thank you, now i finally know why 'it' is used at all :)